v2/azure-arm.yaml

```diff
@@ -1505,16 +1505,58 @@ objectModelConfiguration:
       Redis_PatchSchedule:
         $exportAs: RedisPatchSchedule
         $supportedFrom: v2.10.0
+      RedisCreateProperties_RedisConfiguration:
+        $nameInNextVersion: RedisCommonPropertiesRedisConfiguration
+      RedisCreateProperties_RedisConfiguration_STATUS:
+        $nameInNextVersion: RedisCommonPropertiesRedisConfiguration_STATUS
+      RedisProperties_RedisConfiguration_STATUS:
+        $nameInNextVersion: RedisCommonPropertiesRedisConfiguration_STATUS
+    2024-11-01:
+      Redis:
+        $exportAs: Redis
+        $supportedFrom: v2.16.0
+        $azureGeneratedSecrets:
+          - PrimaryKey
+          - SecondaryKey
+          - HostName
+          - Port
+          - SSLPort
+      RedisCreateProperties:
+        SubnetId:
+          $referenceType: arm
+      Redis_AccessPolicy:
+        $exportAs: RedisAccessPolicy
+        $supportedFrom: v2.16.0
+      Redis_AccessPolicyAssignment:
+        $exportAs: RedisAccessPolicyAssignment
+        $supportedFrom: v2.16.0
+      RedisCacheAccessPolicyAssignmentProperties:
+        ObjectId:
+          $importConfigMapMode: optional
+        ObjectIdAlias:
+          $importConfigMapMode: optional
+      Redis_FirewallRule:
+        $exportAs: RedisFirewallRule
+        $supportedFrom: v2.16.0
+      Redis_LinkedServer:
+        $exportAs: RedisLinkedServer
+        $supportedFrom: v2.16.0
+      RedisLinkedServerCreateProperties:
+        LinkedRedisCacheId:
+          $referenceType: arm
+      Redis_PatchSchedule:
+        $exportAs: RedisPatchSchedule
+        $supportedFrom: v2.16.0
     2025-04-01:
       RedisEnterprise:
         $exportAs: RedisEnterprise
-        $supportedFrom: v2.15.0
+        $supportedFrom: v2.16.0
```

Comment:
Never change the `$supportedFrom` value for an existing resource. Revert this line to `v2.15.0`.